xen/drivers/passthrough/device_tree.c

```diff
@@ -318,6 +319,10 @@ int iommu_do_dt_domctl(struct xen_domctl *domctl, struct domain *d,
             break;
         }
 
+        ret = ac_assign_dt_device(dev, d);
```

Comment:
Commit message does not mention this change. Are you sure it should be in this patch?

Reply:
moved to separate patch